rust/Cargo.lock

```diff
@@ -985,15 +961,15 @@ dependencies = [
  "ff",
  "group",
  "pasta_curves",
- "rand_core 0.6.3",
+ "rand_core",
```

Comment:
Same. I'll stop leaving comments on each line like this now, but there are probably more. Is it an issue?